plugin/pkg/scheduler/factory/factory.go

```diff
@@ -99,7 +99,8 @@ func (f *ConfigFactory) CreateFromKeys(predicateKeys, priorityKeys util.StringSe
 	// Watch minions.
 	// Minions may be listed frequently, so provide a local up-to-date cache.
 	if false {
-		// Disable this code until minions support watches.
+		// Disable this code until minions support watches. Note when this code is enabled,
+		// we need to make sure minion ListWatcher has proper FieldSelector.
```

Comment:
Yes and no.  Basic node watch is implemented in #2662, but it only enables watch by label (no watch by status).

We can either:
1. Watch all nodes, and filter out unhealthy nodes in some custom watch handler.  Not sure if the listwatch pkg support it.
or
2. Do not enable watch, wait until node status is fully understand by the system.